pkgs/tools/X11/xpra/default.nix

```
@@ -111,8 +120,7 @@ in buildPythonApplication rec {
       --set XPRA_XKB_CONFIG_ROOT "${xorg.xkeyboardconfig}/share/X11/xkb"
       --prefix LD_LIBRARY_PATH : ${libfakeXinerama}/lib
       --prefix PATH : ${lib.makeBinPath [ getopt xorgserver xauth which util-linux pulseaudio ]}
-    )
-  '';
+  '' + lib.optionalString withNvenc "--prefix LD_LIBRARY_PATH : ${nvidia_x11}/lib" + ")";
```

```suggestion
  '' + lib.optionalString withNvenc ''
      --prefix LD_LIBRARY_PATH : ${nvidia_x11}/lib
  '' + ''
    )
  '';
```
This should make it a non-rebuld